### DISCUSSION TITLE
Is there a recommended way for community members to give suggestions to the CAB?

### DISCUSSION THREAD

**Discussion by kozo2** [BioconductorBoards/CABWorkingGroups#20] [Q&A]
From https://www.youtube.com/watch?v=LTT1i_ACpxc

**Comment by kozo2** [comment]
We are happy to receive suggestions and feedback and recommend reaching us via email (cab@bioconductor.org) or by messaging us on the #community-advisory-board channel of the Bioconductor community slack (https://bioc-community.herokuapp.com/).

**Comment by lshep** [comment]
See also:   https://github.com/BioconductorBoards/CABWorkingGroups/discussions/16#discussioncomment-1301749
